drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -984,7 +1024,13 @@ function DragAndDropBlock(runtime, element, configuration) {
             data: '{}'
         }).done(function(data){
             state.attempts = data.attempts;
-            state.overall_feedback = data.feedback;
+            state.feedback = data.feedback;
+            state.overall_feedback = data.overall_feedback;
+
+            if (data.feedback) {
+                state.last_action_correct = false;
```

Comment:
@itsjeyd Having thought about this, I believe this is exactly right name for it. `last_action_correct` is used in `itemFeedbackPopupTemplate` to choose popup class - in that case we're showing a popup for last action; checking answer in assessment mode and dropping an item in standard mode.